webapp/i18n/en.json

```diff
@@ -2300,16 +2300,16 @@
   "user.settings.notifications.commentsNever": "Do not trigger notifications on messages in reply threads unless I'm mentioned",
   "user.settings.notifications.commentsRoot": "Trigger notifications on messages in threads that I start",
   "user.settings.notifications.desktop": "Send desktop notifications",
-  "user.settings.notifications.desktop.allFirefoxForever": "For all activity, shown indefinitely",
-  "user.settings.notifications.desktop.allFirefoxTimed": "For all activity, shown for {seconds} seconds",
+  "user.settings.notifications.desktop.allSansSoundForever": "For all activity, shown indefinitely",
```

Comment:
Can you change the translation ids to be like `allSoundHiddenForever`? Translators might have difficulty understanding the "sans" part (unless of course they're translating into French 😄)